packages/aws-cdk-lib/aws-appconfig/test/environment.test.ts

```diff
@@ -54,6 +54,216 @@ describe('environment', () => {
     });
   });
 
+  test('environment with single deployment', () => {
```

Comment:
Can you please add coverage for imported environments (`fromEnvironmentArn` / `fromEnvironmentAttributes`)?